install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -406,7 +406,7 @@ spec:
 {{- end }}
 {{- if and .Values.nodeinit.enabled (not (eq .Values.nodeinit.bootstrapFile "")) }}
       - name: wait-for-node-init
-        command: ['sh', '-c', 'until stat {{ .Values.nodeinit.bootstrapFile }} > /dev/null 2>&1; do echo "Waiting on node-init to run..."; sleep 1; done']
```

Comment:
I think we should keep the `> /dev/null 2>&1`

Reply:
What for? It's there to silence `stat`, which is not needed for `test -s`. It has been removed on master too.

Reply:
I think test didn't do anything with stdout as per my quick test below :) 

```
$ until test -s /invalid/file ; do echo "Waiting on node-init to run..."; sleep 1; done 
Waiting on node-init to run...
Waiting on node-init to run...
...

$ until test -s /invalid/file > /dev/null 2>&1; do echo "Waiting on node-init to run..."; sleep 1; done
Waiting on node-init to run...
Waiting on node-init to run...
Waiting on node-init to run...
...
...